test.sh

```diff
@@ -115,6 +115,19 @@ if [[ "$RUN" =~ "integration" ]] ; then
     python2 test/integration-test.py "${args[@]}"
 fi
 
+# Test that just ./start.py works, which is a proxy for testing that
+# `docker-compose up` works, since that just runs start.py (via entrypoint.sh).
+if [[ "$RUN" =~ "start" ]] ; then
+  ./start.py &
```

Comment:
Feels like we should clean this up when we are done, something like `START_PID=$!;for ...; kill $START_PID`?

Reply:
It gets cleaned up when the container exits. We could be extra tidy in case someone runs test.sh outside a container, but I don't think that's a use case we aim to support.